docs/en/latest/building-apisix.md

```diff
@@ -45,22 +45,22 @@ To build and package APISIX for a specific platform, see [apisix-build-tools](ht
 
 ## Building APISIX from source
 
-Install dependencies using the script provided by APISIX:
+First of all, we need to specify the version `APISIX_ VERSION` to be installed:
 
 ```shell
-curl https://raw.githubusercontent.com/apache/apisix/master/utils/install-dependencies.sh -sL | bash -
+APISIX_VERSION='3.7.0'
 ```
 
-Save the APISIX version to an environment variable to be used next:
+Then, you can run the following command to clone the APISIX source code from Github:
 
 ```shell
-APISIX_VERSION='3.7.0'
+git clone --depth 1 --branch ${APISIX_VERSION} https://github.com/apache/apisix.git apisix-${APISIX_VERSION}
 ```
 
-Clone the APISIX source code of this version into a new directory `apisix-APISIX_VERSION`:
+Now, by changing to the directory of the APISIX source code, you can install the dependencies with the following command:
```

Comment:
I don't see any commands to install dependencies of apisix.

Reply:
Updated!